scipy/stats/tests/test_stats.py

```diff
@@ -8092,45 +7973,21 @@ def test_pval_ci_match(self):
         # 
         # And the corresponding statement for the alternative='greater' case.
 
-        # This test may fail if we the CI bound falls on a value present 
-        # multiple times in the sample. We make sure this does not happen by 
-        # generating samples via permutations rather than pure random.
-
-        for xi in x:
```

Comment:
I misunderstood what was going on:
https://github.com/scipy/scipy/pull/12680#discussion_r1230401636

But I think you'll find the suggested version is a bit simpler. If we want to test for different size samples, we can add, e.g.,  `@pytest.mark.parametrize('size', [23, 85, 129])`.  Where the numbers are arbitrary. I don't think we need 100 of them, though.